src/utils/DynamicPagesFinder.js

```diff
@@ -0,0 +1,35 @@
+// (c) Copyright 2018 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import {UpdateServerPages} from "../pages/ReplaceServer/UpdateServerPages";
+
+const PAGES_HOLDERS = {
+  'servers_ReplaceServer': UpdateServerPages //<menu>_<operation> : <all the possible pages defined>
+};
+
+export function getPages(menuName, operationName, steps) {
```

Comment:
its not at all clear what this class or function are doing... needs comments

Reply:
will update